tests/video/test_loading_cursor.py

```diff
@@ -0,0 +1,9 @@
+from panda3d.core import MovieVideo
+import pytest
+
+
+def test_cursor_check(): 
+    movie_path = "../../samples/media-player/PandaSneezes.ogv"
+    get_File = MovieVideo.get(movie_path)
+    cursor_type = MovieVideo.open(get_File)
+    assert cursor_type != None
```

Comment:
Use `is not None` instead of `!= None` https://stackoverflow.com/questions/2209755/python-operation-vs-is-not